pkgs/development/python-modules/pyopenssl/default.nix

```diff
@@ -26,6 +26,19 @@ buildPythonPackage rec {
     sed -i 's/test_set_default_verify_paths/noop/' tests/test_ssl.py
     # https://github.com/pyca/pyopenssl/issues/692
     sed -i 's/test_fallback_default_verify_paths/noop/' tests/test_ssl.py
+    # https://github.com/pyca/pyopenssl/issues/791
+    sed -i 's/test_op_no_compression/noop/' tests/test_ssl.py
+    sed -i 's/test_npn_advertise_error/noop/' tests/test_ssl.py
+    sed -i 's/test_npn_select_error/noop/' tests/test_ssl.py
+    sed -i 's/test_npn_client_fail/noop/' tests/test_ssl.py
+    sed -i 's/test_npn_success/noop/' tests/test_ssl.py
+    sed -i 's/test_use_certificate_chain_file_unicode/noop/' tests/test_ssl.py
+    sed -i 's/test_use_certificate_chain_file_bytes/noop/' tests/test_ssl.py
+    sed -i 's/test_add_extra_chain_cert/noop/' tests/test_ssl.py
+    sed -i 's/test_set_session_id_fail/noop/' tests/test_ssl.py
+    sed -i 's/test_verify_with_revoked/noop/' tests/test_crypto.py
+    sed -i 's/test_set_notAfter/noop/' tests/test_crypto.py
+    sed -i 's/test_set_notBefore/noop/' tests/test_crypto.py
```

Comment:
Is it possible to disable tests with `pytest` instead?
https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/tools/database/pgcli/default.nix#L17

Reply:
As an alternative with so many test failures, it might be also possible to set `doCheck = openssl != libressl` until it is fixed upstream?

Reply:
Ah, interesting. What would you recommend: should I make the derivation accept `libressl` as well, and do a literal `!=` comparison between the `openssl` and `libressl` attributes?